types/http-aws-es/package.json

```diff
@@ -1,6 +1,7 @@
 {
     "private": true,
     "dependencies": {
-        "aws-sdk": "^2.7.0"
+        "aws-sdk": "^2.7.0",
+        "elasticsearch": "^6.0.0"
     }
```

Comment:
Given:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f3c33f18306eb60c2f76fe7b45e135fe27d0f4b7/types/http-aws-es/index.d.ts#L1

Looking at [TheDeveloper/http-aws-es/v6.0.0/package.json:20-23](https://github.com/TheDeveloper/http-aws-es/blob/v6.0.0/package.json#L20-L23), you probably want to use:

```json
{
  "dependencies": {
    "aws-sdk": "^2.138.0",
    "elasticsearch": "^15.0.0"
  }
}
```

Reply:
Good catch!

Reply:
It turns out I made a mistake in https://github.com/microsoft/DefinitelyTyped-tools/pull/123 .  When Elasticsearch started providing their own definitions they also moved the package from `elasticsearch` to `@elastic/elasticsearch` and reset the major version downwards (but kept the old GitHub repo).  They also rearranged the internals breaking `http-aws-es`.

I still think removing `@types/elasticsearch` is the right call because it's not clear which of `elasticsearch` and `@elastic/elasticsearch` it refers to, has fallen out of date and isn't worth investing in.  But I don't know what to do about `http-aws-es`.

@marcogrcr Thoughts?

Reply:
@pushplay I believe the best course of action is to remove the typings for `http-aws-es` because the package seems to be abandoned. The [last commit](https://github.com/TheDeveloper/http-aws-es/commit/5099705b10afd65944f300f1c3b73025edcc1283) was from May 2018 when the last version ([6.0.0](https://www.npmjs.com/package/http-aws-es)) was released in the `npm` public repo.

Instead, typings should be created for [`aws-elasticsearch-connector`](https://www.npmjs.com/package/aws-elasticsearch-connector) (which actually is a fork from `http-aws-es`). Then, submit a pull request to directly to [compwright/aws-elasticsearch-connector](https://github.com/compwright/aws-elasticsearch-connector) to add TypeScript support, so that ideally, these typings can be removed (just like you're doing now for `elasticsearch`).

Reply:
@marcogrcr Thanks.  Yeah I think that's a good idea.  I'll remove @types/http-aws-es in this PR and add @types/aws-elasticsearch-connector in another.